telemetry/specs/INT.mdk

```diff
@@ -1953,18 +2074,19 @@ INT-MD over IPv4/GRE
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+  |
    |                      queue occupancy of hop1                  |  |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<-+
-   |      Inner IPv4 Header + Payload + Padding (L2/L3/ESP/….)     |
+   |                    Payload (Original Packet)                  |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-## Example with INT-MX over IPv4/GRE
+## Example with INT-MX over IPv4/GRE (Original packet CE or IP)
 
 In this scenario host1 sends an IPv4 packet to host2.
-The ToR switch of host1 (Switch1) acts as the INT source.
-It does IPv4/GRE encapsulation and inserts INT-MX header before the
-inner (original) IPv4. Switch2 processes the INT-MX header. Finally, the ToR
+The ToR switch of host1 (Switch1) acts as the INT source. It add an Ethernet 
+(L2) header, does IPv4/GRE encapsulation and inserts INT-MX header before the
+inner (original) packet. Switch2 processes the INT-MX header. Finally, the ToR
 switch of host2 (Switch3) acts as the INT sink and removes the INT-MX header
-and decapsulates outer IPv4/GRE before forwarding the packet to host2.
+and decapsulates outer L2 header, outer IPv4/GRE before forwarding the packet 
+to host2.
 
 Below is the packet received by INT sink Switch3, starting from the
 outer IPv4 header. The G bit of INT shim is set to 1 to indicate that
```

Comment:
s/outer IPv4 header/outer Ethernet header

Reply:
We would only need to remove the L2 Header, Outer IPv4/GRE and INT. The payload is the original packet. Why decapsulate the outer Ethernet header?

Reply:
The comment was about "starting from the outer IPv4 header", but your figure starts with the outer Ethernet header.

Reply:
Done.